exporter/elasticsearchexporter/model.go

```diff
@@ -321,6 +321,15 @@ func (m *encodeModel) upsertMetricDataPointValue(documents map[uint32]objmodel.D
 	return nil
 }
 
+func summaryToValue(dp pmetric.SummaryDataPoint) (pcommon.Value, error) {
+	// quantiles are ignored
```

Comment:
```suggestion
	// TODO: Add support for quantiles
```
Also, can we create an issue for this?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34561